yt/fields/tests/test_field_access.py

```diff
@@ -28,6 +28,6 @@ def test_field_access():
         oap = ProjectionPlot(ds, [1, 1, 1], field)
 
         for plot_object in [s, oas, p, oap, prof, phase]:
-            plot_object._setup_plots()
+            plot_object.render()
```

Comment:
Just so I am clear, this PR changes it to use the current API, but the old API is still currently in place, right?  This is just keeping up to date with a deprecation?

Reply:
the `_setup_plots` method isn't actually deprecated, I just think it's best to avoid calling private methods directly from outside a class (which is the whole reason I added a public equivalent in this case).
But yes, it's just a no-op change toward stable API